Makefile

```diff
@@ -41,8 +41,7 @@ test: build-test-container test-in-container test-doc-examples
 
 test-in-container:
 	@# use it like this: `make test-in-container TEST_TARGET=tests/integration/test_utils.py`
-	sed -e s#"${HOME}"#/root#g ${HOME}/.kube/config > $(kubedir)/config
-	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v /var/run/docker.sock:/var/run/docker.sock -v $(CURDIR):/src -v $(CURDIR)/pytest-container.ini:/src/pytest.ini -v ${HOME}/.minikube:/root/.minikube -v $(kubedir):/root/.kube $(TEST_IMAGE_NAME) make exec-test TEST_TARGET=$(TEST_TARGET)
+	docker run --net=host --rm -v /dev:/dev:ro -v /var/lib/docker:/var/lib/docker:ro --security-opt label=disable --cap-add SYS_ADMIN -ti -v /var/run/docker.sock:/var/run/docker.sock -v $(CURDIR):/src -v $(CURDIR)/pytest-container.ini:/src/pytest.ini $(TEST_IMAGE_NAME) oc login 127.0.0.1:8443 -u developer -p developer -n conu --insecure-skip-tls-verify && make exec-test TEST_TARGET=$(TEST_TARGET)
```

Comment:
I guess you want ([see also](https://stackoverflow.com/questions/28490874/docker-run-image-multiple-commands))
`docker run ... $(TEST_IMAGE_NAME) /bin/bash -c "oc login ... && make exec-test ..."`